src/instana/__init__.py

```diff
@@ -199,6 +199,7 @@ def boot_agent():
         #     client,  # noqa: F401
         #     server,  # noqa: F401
         # )
+        from instana.instrumentation import pyramid_inst
```

Comment:
Why don't we add it to the other [import block](https://github.com/instana/python-sensor/blob/990be4383df2a3c5cc730ff0309235ed782ed47a/src/instana/__init__.py#L165)?

Reply:
Good catch! Done ✅